app/src/main/java/org/oppia/android/app/hintsandsolution/HintsAndSolutionDialogFragmentPresenter.kt

```diff
@@ -211,7 +211,13 @@ class HintsAndSolutionDialogFragmentPresenter @Inject constructor(
     }
 
     binding.hintTitle.text =
-      resourceHandler.capitalizeForHumans(hintsViewModel.title.get()!!.replace("_", " "))
+      resourceHandler.capitalizeForHumans(
```

Comment:
This part shouldn't be needed anymore since we can rely on capitalization from the string resource itself.

Reply:
Done. The capitalization logic has been removed